client/src/api/eventstream.tsx

```diff
@@ -22,7 +21,7 @@ import {
   ControlPoint,
   Flight,
   FrontLine,
-  IadsConnection,
+//  IadsConnection,
```

Comment:
Prevent eslint warning, will be included back in PR for IADS

Reply:
Just delete commented out code.